lib/ansible/modules/web_infrastructure/ansible_tower/tower_credential.py

```diff
@@ -46,8 +46,20 @@
     kind:
       description:
         - Type of credential being added.  The ssh choice refers to a Tower Machine credential.
-      required: True
+      required: False
       choices: ["ssh", "vault", "net", "scm", "aws", "vmware", "satellite6", "cloudforms", "gce", "azure_rm", "openstack", "rhv", "insights", "tower"]
+    credential_type:
+      description:
+        - Name of credential type.
+      required: False
+      version_added: "2.7"
```

Comment:
Is there any need for this to show up in an earlier version?

Reply:
I think you need to first merge to devel, and then you can backport to stable-2.7.

I think the core team says they don't backport features, so... the backport might not work either.

Reply:
Correct on both counts.  Please close this PR and reopen against the devel branch.